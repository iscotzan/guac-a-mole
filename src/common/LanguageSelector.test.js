import React from "react";
import LanguageSelector from "./LanguageSelector";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

const wrapper = shallow(<LanguageSelector />);
describe("<LanguageSelector />", () => {
  it("should be defined", () => {
    expect(wrapper).toBeDefined();
  });

  it("renders without crashing", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
