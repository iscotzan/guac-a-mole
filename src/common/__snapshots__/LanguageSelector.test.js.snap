// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LanguageSelector /> renders without crashing 1`] = `<[object Object] />`;
